.github/workflows/deploy-version-action.yml

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I no longer see the point of this script as the existing deploy action does the same thing

Reply:
I was also wondering wondering why we need a separate workflow to deploy a new version. Possible we could create one that runs the ansible script before deploying, it would make sense then